cmd/dockerd/daemon_windows.go

```diff
@@ -93,7 +94,30 @@ func newCgroupParent(config *config.Config) string {
 	return ""
 }
 
-func (cli *DaemonCli) initContainerD(_ context.Context) (func(time.Duration) error, error) {
+func (cli *DaemonCli) initContainerD(ctx context.Context) (func(time.Duration) error, error) {
+	if cli.Config.ContainerdAddr == "" && cli.Config.DefaultRuntime == config.WindowsV2RuntimeName {
+		cli.Config.ContainerdAddr = supervisor.GrpcPipeName
```

Comment:
Curious about the;

```go
cli.Config.ContainerdAddr = supervisor.GrpcPipeName
```

Wouldn't that be set as part of `setDefaults()` ? https://github.com/moby/moby/blob/c8fc98937d7bb75093e3eaeb0d9fd6b717e46bdc/libcontainerd/supervisor/remote_daemon_windows.go#L14

Or was this specifically for the `initContainerdRuntime()`?